### DISCUSSION TITLE
v2.7.1

### DISCUSSION THREAD

**Discussion by timmo001** [timmo001/system-bridge#866] [General]
## 🐛 Bug fixes

- Fix startup crashing @timmo001 (#860)
- Fix Windows tray crashes outside of installed dir @timmo001 (#862)

<hr /><em>This discussion was created from the release <a href='https://github.com/timmo001/system-bridge/releases/tag/v2.7.1'>v2.7.1</a>.</em>

**Comment by SilentException** [comment]
Hi, this is actually related to v2.7.2 but since there is no discussion for that release yet I'll just leve it here :)
v2.7.2 is internally labelled as 2.7.1 so even though I have 2.7.2 installed, in the tray and in the settings there is a notification that update is available.

**Reply by timmo001** [reply]
Yeah, I didn't update the package version. The changes are there though. The next update will fix this
